insights: Filtering by context should OR together multiple repos (#45574)
sourcegraph/sourcegraph: CHANGELOG.md

- Code Insights: fixed an issue where certain queries matching sequential whitespace characters would overcount. [#44969](https://github.com/sourcegraph/sourcegraph/pull/44969)
- GitHub fine-grained Personal Access Tokens can now clone repositories correctly, but are not yet officially supported. [#45137](https://github.com/sourcegraph/sourcegraph/pull/45137)
- Detect-and-track Code Insights will now return data for repositories without sub-repo permissions even when sub-repo permissions are enabled on the instance. [#45631](https://github.com/sourcegraph/sourcegraph/pull/45361)

> - An issue with insights that excluded data when the insight was filtered by a search context that included multiple repositories. [#45574](https://github.com/sourcegraph/sourcegraph/pull/45574)

Reviewer: ```suggestion
- Code Insights: fixed an issue where filtering by a search context that included multiple repositories would exclude data. [#45574](https://github.com/sourcegraph/sourcegraph/pull/45574)
```
